### PR TITLE
New Overview Page

### PR DESCRIPTION
**Pull request by ryandurfey**
Updated network diagram.
Updated intro text slightly.
Added list of systems with links to documentation.

### CODE REVIEW

**Comment by PSUdaemon** [comment]
It would be really cool if we could figure out how to use SVGs instead of PNGs.

File: docs/source/overview/introduction.rst [hunk 1]
```diff
@@ -15,14 +15,34 @@
...
-.. image:: traffic_control_overview_3.png
+.. image:: CDN-Traffic-Control.png
 	:align: center
 
+
+**Traffic Ops**
+  * `Traffic Ops <http://trafficcontrol.apache.org/docs/latest/overview/traffic_ops.html/>`_ is used to configure caching  servers and CDN delivery services. It also contains APIs used to access CDN data.
+
+**Traffic Router**
+  * `Traffic Router <http://trafficcontrol.apache.org/docs/latest/overview/traffic_ops.html/>`_ is used to route clients requests to the closest healthy cache by analyzing the health, capacity, and state of the caching servers and relative distance from each cache group to the location of the client.
```
Comment (mitchell852) [inline docs/source/overview/introduction.rst]: copy / paste error on the link?

**Comment by mitchell852** [comment]
no big deal but in the future, descriptive commit messages can be very helpful. i see a few commit messages in this PR that aren't too descriptive of the change (Update introduction.rst) but I get it, you are rewriting documentation so maybe descriptive commit messages are a little harder to come by...

**Comment by ryandurfey** [comment]
Thanks for the feedback guys!  Yep, that was a copy/paste error on the link.  I will work on detailing  my commit messages.

File: docs/source/overview/introduction.rst (hunk 1 above)
Comment (mitchell852) [inline docs/source/overview/introduction.rst]: i can merge this PR but it looks like this copy/paste error is still there

**Pull request closed without merging**